service/src/test/java/bio/terra/pipelines/testutils/MockMvcUtils.java

```diff
@@ -47,12 +49,15 @@ public static String convertToJsonString(Object obj) throws JsonProcessingExcept
           new BearerToken(UUID.randomUUID().toString()));
 
   // Pipelines test constants
-  public static final String TEST_PIPELINE_ID_1 = "test-pipeline-id-1";
-  public static final String TEST_PIPELINE_VERSION_1 = "1.0.0";
-  public static final String TEST_PIPELINE_NAME_1 = "Test Pipeline Name One";
-  public static final String TEST_PIPELINE_DESCRIPTION_1 = "Test Pipeline Description One";
-  public static final String TEST_PIPELINE_ID_2 = "test-pipeline-id-2";
-  public static final String TEST_PIPELINE_VERSION_2 = "1.0.0";
+  public static final String TEST_PIPELINE_ID_1 =
```

Comment:
nice grouping of commonly used test utils.  Was thinking maybe we want to rename the file or split the mvc stuff from like general `TestUtil` stuff but not something we have to decide now, just ToL

Reply:
i like it. easy enough to refactor now.